instrumentation/log4j/log4j-2.13.2/library/src/main/java/io/opentelemetry/instrumentation/log4j/v2_13_2/OpenTelemetryContextDataProvider.java

```diff
@@ -47,9 +47,11 @@
 
     Map<String, String> contextData = new HashMap<>();
     SpanContext spanContext = currentSpan.getContext();
-    contextData.put("traceId", spanContext.getTraceId().toLowerBase16());
-    contextData.put("spanId", spanContext.getSpanId().toLowerBase16());
-    contextData.put("traceFlags", spanContext.getTraceFlags().toLowerBase16());
+    contextData.put("traceId", spanContext.getTraceIdAsHexString());
+    contextData.put("spanId", spanContext.getSpanIdAsHexString());
+    if (spanContext.isSampled()) {
+      contextData.put("sampled", "true");
```

Comment:
Figured this is more informative than even what we were doing previously of just putting a hex string of a byte